src/zkp/pienc.rs

```diff
@@ -25,13 +25,13 @@
 //! [^cite]: Ran Canetti, Rosario Gennaro, Steven Goldfeder, Nikolaos Makriyannis, and Udi Peled.
 //! UC Non-Interactive, Proactive, Threshold ECDSA with Identifiable Aborts.
 //! [EPrint archive, 2021](https://eprint.iacr.org/2021/060.pdf).
-
+#[allow(unused_imports)]
```

Comment:
question: is this still needed? Can you just delete the imports that aren't used?

If you import stuff that's only used in tests, you can pull that into it's own line and cfg-test it, like
```rust
#[cfg(test)]
use crate::utils::k256_order;
```